optimize/backend/pom.xml

```diff
@@ -398,6 +398,21 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>camunda-service</artifactId>
+      <version>${project.version}</version>
+      <exclusions>
```

Comment:
These exclusions were added to fix build issues. There were errors related to not being able to find the libraries

Reply:
I just removed this. After chatting with Mattia, Optimize cannot use any monorepo dependencies until it is part of the single application